TwoPaneLayout/library/src/main/java/com/microsoft/device/display/twopanelayout/TwoPaneLayout.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.twopanelayout
+
+import android.graphics.Rect
+import androidx.compose.foundation.layout.LayoutScopeMarker
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.Layout
+import androidx.compose.ui.platform.LocalView
+import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.microsoft.device.display.twopanelayout.screenState.ConfigScreenState
+import com.microsoft.device.display.twopanelayout.screenState.ScreenStateViewModel
+
+@Composable
+inline fun TwoPaneLayout(
```

Comment:
Please, since you are creating a library with a public API consider follow KDoc so we later can generate its public API doc.

This comment is to be applicable on all public methods you expose for devs to use the library.

Reply:
Do we have other public methods that developer will have to use or that are accessible from outside?
Wondering about this: https://github.com/microsoft/surface-duo-compose-samples/blob/39816085a93da4a4de0ab35a4d2d2d18055a697a/TwoPaneLayout/library/src/main/java/com/microsoft/device/display/twopanelayout/TwoPaneLayoutImpl.kt#L25 and others?

Reply:
No, `TwoPaneLayout` should be the only one public API for the developer to use. The reason of `@PublishedApi` for `twoPaneMeasurePolicy` is because `Public-API inline function cannot access non-public-API 'internal fun twoPaneMeasurePolicy`

Reply:
but then, does it mean that `twoPaneMeasurePolicy` is accessible from outside? I understand the motivation/requirements for that, but if you need it, I think that you can do to avoid exposing it to the public is moving the code of this class to the `TwoPaneLayout` class and make that api private to the component.

Exposed functions in the public API doesn't not stop anyone to try to use these functions, which can cause the problems while using the component or or making devs confused about how they should use the component.

Reply:
`@PublishedApi internal` is the intended way of exposing non-public API for using in public inline functions. That `@PublishedApi internal` member("twoPaneMeasurePolicy") becomes effectively public and its name doesn't get mangled.